### ISSUE TITLE
SeqBuddy print() function needs to be __str__

### ISSUE THREAD

**Issue by etiology** [biologyguy/BuddySuite#27]
SeqBuddy is [defining it's own print behavior](https://github.com/biologyguy/BuddySuite/blob/78b6688eff983c95eca27411d818970a572683fe/workshop/SeqBuddy.py#L272) and this should use the built in functions.  This class should make use of the ```__str__``` and ```__repr__``` functions.

The ```__str__``` function looks like it is doing too much.  This needs to be cleaned up.

Please place these directly under the __init__ of SeqBuddy.

**Comment by biologyguy** [comment]
Not sure why I had the print() method in there, given that it isn't actually used anywhere in the code... Deleted!
I've also moved all the dunder methods just below __init__ as recommended.
As for it doing too much, it's delicate. Because there are so many file formats being supported, most with their own little quirks, __str__ has sort of grown. You'll notice that most of the logic is essentially a switch to apply format specific tweeks to the output.
I've made similar changes to AlignBuddy and PhyloBuddy, but Database buddy will require more consideration. The print() method is very large, so will probably need just be renamed to something else.
